capstone/src/main/webapp/announcements.html

```diff
@@ -4,8 +4,9 @@
     <meta charset="UTF-8">
     <title>Announcements</title>
     <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+    <script src="script.js"></script>
+    <div id="top-navigation"></div>
```

Comment:
I thought even header divs weren't supposed to go in the <head> section...maybe I'm wrong though?

Reply:
Yeah I think you're right according to this: https://www.w3schools.com/tags/tag_head.asp#:~:text=The%20element%20is%20a,scripts%2C%20and%20other%20meta%20information. The script tag should be fine but I moved the div tag to the body in the other files as well, thanks!